frontend/src/components/SideNav.tsx

```diff
@@ -195,6 +195,7 @@ export default class SideNav extends React.Component<SideNavProps, SideNavState>
 
     this.state = {
       collapsed,
+      // Set jupyterHubAvailable to false so UI don't show Jupyter Hub link
```

Comment:
Why not just remove the references entirely?

Reply:
I believe we are going to hyperlink the new Jupyter UI in future

Reply:
Can I ask why are we removing the link? What I did for my deployment was just to add an ambassador redirect to my actual jhub url, whether it is the classic UI or the new UI.

Reply:
we are not entirely sure what the plan is. it can still be an entrypoint for new ui, or jaas on gcp. it's safer to keep it but just my opinion.